crates/auth/src/token.rs

```diff
@@ -11,6 +11,7 @@ use std::sync::Arc;
 
 #[derive(Debug)]
 pub struct Token {
+    #[expect(clippy::struct_field_names)]
```

Comment:
I don't like this lint. Overlapping struct and field names are too common to have a lint for it and it adds a unnecessary line of code.

Reply:
sure, I can disable it crate-wide.

In this specific case I would suggest renaming it to `value` -- to make usage cleaner:

```rust
let token = Token::new(...);
if token.value == '...' {
  ...
}
```

Or it could be `.value()` with an accessor function. Otherwise it would be `token.token`.